tests/tests_unit/test_db/test_db_reader_unit/test_iceberg_reader.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+from onetl.connection import Iceberg
+from onetl.db import DBReader
+
+pytestmark = pytest.mark.iceberg
```

Comment:
As there is no `hive`, `hdfs` or `s3` mark, but CI always run tests with a specific mark, these tests are not running in CI. Let's copy `test-hive.yml` and `.github/workflows/data/hive` with replacing `hive` with `iceberg`